### DISCUSSION TITLE
Can PHP-requirement since 1.9 version be circumvented?

### DISCUSSION THREAD

**Discussion by markum** [YOURLS/YOURLS#3392] [Q&A] [answered]
I just wanted to update YOURLS from 1.8 to 1.9. Unfortunately I had to downgrade again, because ower webhost still runs on a php version smaller then 7.4 I asked them, if they could provide a newer one. But this has to wait until they upgrade their Debian version, which will not happen before the end of the year.  It is a small provider where we have a personal connection, so we can't and wont put in the effort to change the provider because of this issue. So the question is: Is the minimum php-requirement since 1.9 really necessary and if not, (how) can it be circumvented?

**Comment by LeoColomb** [comment, marked as answer]
Yes, the minimum PHP requirement is really necessary, it is a requirement.
But it should be a requirement for your webhost as well, as older version than 7.4 (as of today) are not supported by the PHP Foundation and must be considered as risky: https://www.php.net/supported-versions.php
Said differently, it's not a requirement for YOURLS features only, but a requirement for overall hosting security.

We'd highly recommend to switch to an up to date webhost.
